content/docs/querying/operators.md

```diff
@@ -25,13 +31,13 @@ value of every data sample in the vector. E.g. if a time series instant vector
 is multiplied by 2, the result is another vector in which every sample value of
 the original vector is multiplied by 2.
 
-**Between two instant vectors**, a binary arithmetic operator only applies to
-vector elements that have identical sets of labels between the two vectors.
-Vector elements that don't find an exact label match on the other side get
-dropped from the result. The metric name of the result vector is carried over
-from the left hand side of the expression.
+**Between two instant vectors**, a binary arithmetic operator is applied to
+each entry in the left-hand-side vector and its [matching element](#vector-matching)
+in the right hand side. The result is carried over to the element to the left
```

Comment:
s/in the right hand side/on the right hand side/

(if it said "in the right hand side vector", "in" would be ok, but if it's just "on the right hand side", it's on)